ddtrace/appsec/CMakeLists.txt

```diff
@@ -4,13 +4,14 @@ include(ExternalProject)
 
 ExternalProject_Add(libddwaf
     GIT_REPOSITORY https://github.com/DataDog/libddwaf.git
-    GIT_TAG 75c23cefb89e7fdf2ecbde494b23ec99ce0d8c3e
+    GIT_TAG 6609da85ceb8850c2dd430da72f53b0cceafb326
```

Comment:
do we need to update the tag here?

Reply:
So it points to the commit that [introduces this option](https://github.com/DataDog/libddwaf/pull/108), otherwise it won't work